### DISCUSSION TITLE
How to properly set up jan to read .pdf files (and optionally images)?

### DISCUSSION THREAD

**Discussion by vico93** [janhq/jan#4173] [Get Help] [answered]
Basically, i tried the Llama 3.2 3B Instruct Q8 model, althrough i have the paperclip enabled for documents (for images it only says "This feature only supports multimodal models"), when i try to attach a .pdf (that was sucessfully summarized by ChatGPT) the response is always something in the lines of "I'm a large language model and don't have capability to access attached files".

Is even possible to configure Jan to do this kind of task?

**Comment by brunoalbertopeixoto** [comment]
> Basically, i tried the Llama 3.2 3B Instruct Q8 model, althrough i have the paperclip enabled for documents (for images it only says "This feature only supports multimodal models"), when i try to attach a .pdf (that was sucessfully summarized by ChatGPT) the response is always something in the lines of "I'm a large language model and don't have capability to access attached files".
> 
> Is even possible to configure Jan to do this kind of task?

I asked your question. This function is in LLM Studio. Something is missing to be installed, perhaps a code to extract text from the PDF. If you describe it, let us know.

**Comment by TC117** [comment, marked as answer]
I think you need to turn this setting on. Working on my end. 
![image](https://github.com/user-attachments/assets/60a4cd14-71af-475c-8c78-3ead41aeb563)


Hope this help
